docs/DebuggingNumericalError.md

```diff
@@ -100,3 +100,28 @@ To print a message followed by one value, inject the following code (where `val`
 ```code
 create.krnl.printf("inputElem: ", val, valType);
 ```
+
+## Finding memory errors
+
+If you know, or suspect, that an onnx-mlir-compiled inference executable
+suffers from memory allocation related issues, the
+framework(https://valgrind.org/) or mtrace memory
```

Comment:
Please use link syntax in markdown, e.g. `[framework](https://valgrind.org/)`. The same for the other urls.

Reply:
done. Thank you!